openmdao/error_checking/check_config.py

```diff
@@ -155,7 +155,14 @@ def _get_used_before_calc_subs(group, input_srcs):
         A dict mapping names of target Systems to a set of names of their
         source Systems that execute after them.
     """
-    sub2i = {sub.name: i for i, sub in enumerate(group._subsystems_allprocs)}
+    sub2i = {}
+    parallel_connections = {}
+    for i, sub in enumerate(group._subsystems_allprocs):
+        if hasattr(sub, '_mpi_proc_allocator') and sub._mpi_proc_allocator.parallel:
+            parallel_connections.update({sub.name: sub.nonlinear_solver.SOLVER})
```

Comment:
It's not clear why this is called `parallel_connections` when it's a dict that maps a parallel group's name to its solver name.